parser_library/src/workspaces/workspace.cpp

```diff
@@ -518,7 +518,7 @@ void workspace::find_and_add_libs(const utils::resource::resource_location& root
             if (processed_canonical_paths.contains(subdir_canonical_path))
                 continue;
 
-            dirs_to_search.emplace_back(std::move(subdir_canonical_path), std::move(subdir));
+            dirs_to_search.emplace_back(std::move(subdir_canonical_path), subdir.lexically_normal());
```

Comment:
will this work if my config contains e.g. "${workspaceFolder}/../common/**"?

Reply:
Probably not. I'll add a test and a possible fix